album/references-beyond-homestuck.yaml

```diff
@@ -6408,6 +6408,14 @@ Duration: '2:42'
 URLs:
 - https://www.youtube.com/watch?v=T6CK7ctyDz4
 ---
+Track: Main Theme (Wii Shop Channel)
+# No official release
```

Comment:
::::p

Reply:
Honestly don't even know what that's doing there, there have been official releases, just not localized